distributed/scheduler.py

```diff
@@ -4913,8 +4916,11 @@ async def update_graph(
                     # (which may not have been added to who_wants yet)
                     client=client,
                 )
-        end = time()
-        self.digest_metric("update-graph-duration", end - start)
+        finally:
```

Comment:
Just noting that this `finally` block is being used to ensure we _always_ decrement the `_active_graph_updates` counter